arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json

```diff
@@ -0,0 +1,14114 @@
+          },
+          {
+            "name": "networkName",
+            "in": "path",
+            "description": "Primary network name.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/NetworkMappingCollection"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets all the network mappings under a network.": {
+            "$ref": "../examples/ReplicationNetworkMappings_ListByReplicationNetworks.json"
+          }
+        }
+      }
+    },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/renewCertificate": {
+      "post": {
+        "tags": [
+          "ReplicationFabrics"
+        ],
+        "summary": "Renews certificate for the fabric.",
+        "description": "Renews the connection certificate for the ASR replication fabric.",
+        "operationId": "ReplicationFabrics_RenewCertificate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "fabric name to renew certs for.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "renewCertificate",
+            "in": "body",
+            "description": "Renew certificate input.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RenewCertificateInput"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Fabric"
+            }
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "deprecated": false,
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Renews certificate for the fabric.": {
+            "$ref": "../examples/ReplicationFabrics_RenewCertificate.json"
+          }
+        }
+      }
+    },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/replicationvCenters/{vCenterName}/operationresults/{jobName}": {
```

Comment:
unless there's a specific reason to include them, we're generally not including them. For long running operations, the sdk generated code will provide methods that can poll and obtain the results as appropriate.